quickstart.md

```diff
@@ -132,23 +132,29 @@ kubectl get runtimeclass
 ```
 Output:
 ```
-NAME            HANDLER         AGE
-kata            kata            9m55s
-kata-clh        kata-clh        9m55s
-kata-clh-tdx    kata-clh-tdx    9m55s
-kata-qemu       kata-qemu       9m55s
-kata-qemu-tdx   kata-qemu-tdx   9m55s
-kata-qemu-sev   kata-qemu-sev   9m55s
+NAME                 HANDLER              AGE
+kata                 kata-qemu            8d
+kata-clh             kata-clh             8d
+kata-qemu            kata-qemu            8d
+kata-qemu-coco-dev   kata-qemu-coco-dev   8d
+kata-qemu-sev        kata-qemu-sev        8d
+kata-qemu-snp        kata-qemu-snp        8d
+kata-qemu-tdx        kata-qemu-tdx        8d
+
 ```
 
 Details on each of the runtime classes:
 
-- *kata* - standard kata runtime using the QEMU hypervisor including all CoCo building blocks for a non CC HW
+- *kata* - standard kata runtime providing a link to kata-qemu
```

Comment:
@fidencio can correct me if I'm wrong, but I think there is a bit of nuance here in that `kata` points to whatever is the default-shim. The default default-shim is kata-qemu, but I'm not sure if we expose the ability to update this in the CoCo operator? 

It might be that it isn't worth going into in this quickstart guide, but we could just reference the handler here and say something like
"Convenience runtime that uses the handler of the default runtime" to cover us?

Reply:
Updated